src/components/ListWraper/ListWraper.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ListItem from './ListItem/ListItem';
+
+const ListWrapper = props => (
```

Comment:
jesli przychodza propy to od razu destrukturyzuj. 
Unikasz dzieki temu powtarzania struktury props.costam, props.costam 

`const ListWrapper = ({ items }) => `